File: C#Part1-Homework/03.OperatorsAndExpressions-Homework/01.OddOrEvenIntegers/OddOrEven.cs
namespace OddOrEvenIntegers
{
    using System;

    class OddOrEven
    {
        // Write an expression that checks if given integer is odd or even.

        static void Main()
        {
            Console.WriteLine("Write number");
            string enter = Console.ReadLine();
            int number = int.Parse(enter);
            bool result = number % 2 != 0;
            if (number % 2 == 0)
            {
                Console.WriteLine("The number is even");
            }
            else
            {
                Console.WriteLine("The number is odd");
            }
            Console.WriteLine(result);
        }
    }
}

File: JavaScript/Homework/Operators and Expressions - Homework/Problem 1 Odd or Even/OddOrEven.js
// Write an expression that checks if given integer is odd or even.

var num;
function isOdd(num) {
    return (num % 2 === 0) ? true : false;
}

for (var i = -5; i <= 5; i++) {
    console.log("Is " + i + " odd: " + isOdd(i));
}